CustomerData/AlmaSection.php

```diff
@@ -60,7 +60,12 @@ public function getSectionData(): array
         $areMergePaymentMethods = $this->checkoutConfigHelper->getAreMergedPaymentMethods();
         $eligibilities[Eligibility::MERGED_TYPE] = $this->eligibility->getEligiblePlans();
         if(!$areMergePaymentMethods){
-            $eligibilities = $this->eligibility->sortEligibilities($eligibilities[Eligibility::MERGED_TYPE]);
+            $enabledPlanInBO = $this->eligibility->getEnabledConfigPaymentPlans();
```

Comment:
I need to speak with you about what is the `MERGED_TYPE` and how your modification fix the error 🧐